deploy/21_hlg.ts

```diff
@@ -65,14 +65,28 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
     process.exit();
   };
 
+  let tokenAmount: BigNumber = BigNumber.from('100000000');
+  let targetChain: BigNumber = BigNumber.from(network.chain);
+  let tokenRecipient: string = deployer.address;
+
   // Future Holograph Utility Token
```

Comment:
Why are the tokens minted in both the `02_sources` script and in this script?

Reply:
`02_sources` script is only used to derive future address for configurations, while the `21_hlg` script does the actual deployment.

Reply:
aha got it